app/models/taxon_lineage.rb

```diff
@@ -23,6 +23,8 @@ class TaxonLineage < ApplicationRecord
     superkingdom: MISSING_SUPERKINGDOM_ID
   }.freeze
 
+  LEVEL_2_NAME = TaxonCount::NAME_2_LEVEL.invert
```

Comment:
maybe this should be next to NAME_2_LEVEL?

Reply:
agreed~

Reply:
ok